Ivanti/pulse-connect/ingest/parser.yml

```diff
@@ -0,0 +1,36 @@
+name: Pulse Connect Secure
+pipeline:
+  - name: kv_event
+    external:
+      name: kv.parse-kv
+      properties:
+        item_sep: '\s'
+  - name: field_extraction
+  - name: msg_parsing
+    external:
+      name: grok.match
+      properties:
+        input_field: '{{kv_event.message.msg}}'
+        output_field: details
+        pattern: '(AUT31556: Unauthenticated request url %{URIPATHPARAM:url} %{GREEDYDATA}|%{GREEDYDATA})'
+      filter: "{{field_extraction.action.name == 'AUT31556'}}"
+  - name: url_extraction
+stages:
+  field_extraction:
+    actions:
+      - set:
+          action.name: "{{kv_event.message.msg.split(':')[0]}}"
```

Comment:
Couldn’t we store this information in an official ECS field? We could probably store this information in `event.code` (and maybe also keep this information in `action.name` during the transition).